src/QuadVelocityController.cpp

```diff
@@ -129,77 +128,38 @@ bool QuadVelocityController::update(const ros::Time& time,
 
 // Get a valid transform, blocks until the transform is received or
 // the request takes too long and times out
-bool QuadVelocityController::getTransformAfterTime(
-        const ros::Time& time,
-        geometry_msgs::TransformStamped& transform,
-        const ros::Time& latest_time_allowed) {
+bool QuadVelocityController::waitForTransform(geometry_msgs::TransformStamped& transform) {
 
     // Catch TransformException exceptions
     try
     {
+        ros::Time time_at_start = ros::Time::now();
```

Comment:
I really think this should be the timestamp of the last transform we processed.  Otherwise we risk outputting at lower frequency than the incoming data